xchange-blockchain/src/main/java/org/knowm/xchange/blockchain/dto/Txs.java

```diff
@@ -122,4 +122,22 @@ public int getVoutSz() {
 
     return this.voutSz;
   }
+
+  @Override
+  public String toString() {
+    return "Txs{" +
+            "blockHeight=" + blockHeight +
+            ", hash='" + hash + '\'' +
+            ", inputs=" + inputs +
+            ", out=" + out +
+            ", relayedBy='" + relayedBy + '\'' +
+            ", result=" + result +
+            ", size=" + size +
+            ", time=" + time +
+            ", txIndex=" + txIndex +
+            ", ver=" + ver +
+            ", vinSz=" + vinSz +
+            ", voutSz=" + voutSz +
+            '}';
```

Comment:
do we relay need those toString for DTOs?
ones like this are to long for both debugging and logging
plus they are similar to JSON to which this object can be serialized to

Reply:
I think they're OK. It's much easier to use `toString` when developing/debugging than to use Jackson deserialization. Much less mental effort plus the who library has these.

Reply:
ok

Reply:
although since Lombok is now part of the library, maybe it could be leveraged somehow automatically to create the `toString` method. (???)

Reply:
well we could, we could also entertain makarids original idea to just serialize the object to JSON in toString - https://github.com/knowm/XChange/issues/2943 - but using our https://github.com/knowm/XChange/blob/develop/xchange-core/src/main/java/org/knowm/xchange/utils/ObjectMapperHelper.java

lets leave it as is in this PR and move the discussion to https://github.com/knowm/XChange/issues/2966